src/main/java/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction.java

```diff
@@ -99,11 +104,15 @@ public String getUrlName() {
 
   @Exported
   public List<LockableResource> getResources() {
-    return LockableResourcesManager.get().getResources();
+    synchronized(lrm) {
+      return new ArrayList<>(lrm.getResources());
+    }
   }
 
   public LockableResource getResource(final String resourceName) {
```

Comment:
check where it is used and set it to deprecated
This operation will be very slowly (because of synchronized over all resources)